build_cpython_macos.rst

```diff
@@ -17,7 +17,7 @@ Get tools and dependencies
 * Install OpenSSL: ``brew install openssl``
 * Install readline: ``brew install readline``
 * Clone the `GitHub cpython project <https://github.com/python/cpython/>`_: ``git clone
-  https://github.com/python/cpython.git`` which creates a ``cpython/`` directory.
+  git@github.com:python/cpython.git`` which creates a ``cpython/`` directory.
```

Comment:
Why do you want to use the Git protocol rather than the HTTPS protocol? Sometimes, the Git protocol is blocked in some networks (public wifi). Developers should not push to this repository but their own fork.

Reply:
Well just a question: why not include fork the repo then download your fork instructions?

Reply:
The fork is explained later, to keep the first section as simple as possible:
https://cpython-core-tutorial.readthedocs.io/en/latest/pr.html#publish-your-pull-request

Reply:
Ok if you say so but it will kind of create some confusions the no git links and the way the fork is explained

Closing the PRs